coalib/results/Diff.py

```diff
@@ -244,10 +243,9 @@ def original(self):
         """
         Retrieves the original file.
         """
-        return self._file
+        return self._generate_linebreaks(self._file)
```

Comment:
You are right. I didn't know about the `lru_cache` thing, seems handy. 
Which approach do you recommend?